src/Resources/translations/SonataAdminBundle.en.xliff

```diff
@@ -74,6 +74,10 @@
                 <source>link_add</source>
                 <target>Add new</target>
             </trans-unit>
+            <trans-unit id="link_edit">
+                <source>link_edit</source>
+                <target>Edit</target>
```

Comment:
`Bearbeiten` for the german translation @kunicmarko20

Reply:
@greg0ire what about french?
@jordisala1991 what about spanish?

Reply:
`Modifier` 🇫🇷

Reply:
Fixed commit message and added translations.

Reply:
@kunicmarko20 `Editar` 🇪🇸